packages/vue/src/components/organisms/SfFooter/SfFooter.js

```diff
@@ -44,6 +44,7 @@ export default {
   },
   methods: {
     toggle(payload) {
+      if(!this.isMobile) return;
```

Comment:
Do you guys have eslint/prettier? Looks like a space is missing
```suggestion
      if (!this.isMobile) return;
```

Reply:
Yes, we have eslint and prettier, thanks :)